sdk/cosmos/azure-cosmos-spark_3-1_2-12/src/main/scala/com/azure/cosmos/spark/CosmosCatalog.scala

```diff
@@ -787,6 +797,12 @@ class CosmosCatalog
   // scalastyle:on cyclomatic.complexity
   // scalastyle:on method.length
 
+  private def redactAuthInfo(cfg: Map[String, String]): Map[String, String] = {
+    cfg.filter((kvp) => !CosmosConfigNames.AccountEndpoint.equalsIgnoreCase(kvp._1) &&
```

Comment:
nit: Could we add some UTs to confirm behaviors like this?

Reply:
good point let me add this because it ultimately is security relevant - to void master key in clear text somewhere in hdfs file system. I will add a step in the existing unit test reading the text file generated simply as text to validate that the test masterkey is not contained. That way we prevent this from happening in some other property as well.